### ISSUE TITLE
Automerge feature for edit

### ISSUE THREAD

**Issue by clintnetwork** [solana-labs/token-list#4081]
Hi,

Please support automerge feature for edits, by example If I want to make one change on my token (if I own the repository for sure) I should be able to do it.

Thanks

**Comment by SolDev-stack** [comment]
Please read the [SPL-token README](https://github.com/SolDev-stack/token-list/blob/main/README.md), and ensure your SPL-token info uses the correct [Uniswap format](https://github.com/SolDev-stack/token-lists/blob/main/README.md).

**Comment by leoluk** [comment]
The problem with edits is proving ownership - this is why edits need manual approval for now.

Discussions are in progress on how to best handle this going forward

**Comment by clintnetwork** [comment]
Yes yes I understand, but just owning repository should be good, I mean if you listed the coin originally its a good ownership proof!

**Comment by clintnetwork** [comment]
Maybe give an automerge result directly after commit and proceed them with CRON, because we have to wait 1 hour to check if we did any mistake.
